trainee_template/react_series.amlg

```diff
@@ -926,6 +973,14 @@
 			)
 		)
 
+		(if (and output_details (contains_index series_detail_values_map "generate_attempts"))
+			;append the amount of regenerate_attempts+1 (regenerate_attempts is zero based)
+			(assign (assoc
+				series_detail_values_map
+					(append series_detail_values_map (assoc "series_generate_attempts" (+ regenerate_attempts 1) ))
```

Comment:
i suggest renaming 'regenerate_attempts' to 'series_generate_attempts' or 'series_generate_retries'

Reply:
done!